pkg/component/component.go

```diff
@@ -320,7 +320,7 @@ func GetCurrent(client *occlient.Client, applicationName string, projectName str
 // During copying binary components, path represent base directory path to binary and files contains path of binary
 // During copying local source components, path represent base directory path whereas files is empty
 // During `odo watch`, path represent base directory path whereas files contains list of changed Files
-func PushLocal(client *occlient.Client, componentName string, applicationName string, path string, out io.Writer, files []string) error {
+var PushLocal = func(client *occlient.Client, componentName string, applicationName string, path string, out io.Writer, files []string) error {
```

Comment:
Finding it hard to mock this function due to :
https://github.com/redhat-developer/odo/pull/820/files#diff-692effa1b0b7a9ed8c1f7bb49a977b9aR363 -> https://github.com/redhat-developer/odo/blob/master/pkg/occlient/occlient.go#L2299

Reply:
don't worry about this one. Lets just make sure that we have e2e tests that covers this

Reply:
Ack fixed it as per your suggestion during our discussion

Reply:
We can change this back to a regular function definition.

Reply:
Aah yes I somehow missed it :(